### ISSUE TITLE
Add accessibility attributes to Autocomplete

### ISSUE THREAD

**Issue by LoremFooBar** [skeletonlabs/skeleton#2536]
### Describe the feature in detail (code, mocks, or screenshots encouraged)

I noticed the current autocomplete implementation is missing accessibility features, such as `role` and `aria-selected` attributes, unlike `ListBox` and `ListBoxItem` which have similar presentation and do provide these attributes. I know v2 is not a priority at the moment, but as this is a relatively simple enhancement maybe it can make it into v2. I'd be happy to try and make the PR myself.

### What type of pull request would this be?

Enhancement

**Comment by endigo9740** [comment]
@LoremFooBar we would welcome a PR for this, otherwise we can certainly aim to address this as part of v3. Thanks for the heads up!

**Comment by endigo9740** [comment]
No activity on this so I'm going to consolidate it this under the v3 effort:

https://github.com/skeletonlabs/skeleton/issues/2366
